mqtt-gpio-monitor.py

```diff
@@ -189,13 +190,16 @@ def on_message(mosq, obj, msg):
 
     if GPIO_MODULE:
         if pin not in GPIO_OUTPUT_PINS:
-            GPIO.setup(pin, GPIO.OUT, initial=GPIO.HIGH)
-            GPIO_OUTPUT_PINS.append(pin)
-
-        if value == 1:
```

Comment:
You are no longer actually setting the pin state, from what I can see.

Reply:
You would be right.  I'm trying to test the gpio through my piface and I think it's allowing it to work without that.  Let me see if I can fix it quickly.